apps/condo/domains/common/utils/serverSchema/accessUtils.js

```diff
@@ -0,0 +1,55 @@
+const { get } = require('lodash')
+
+/**
+ * Check whether user is support or user and it has access for requested list and fields
+ * @returns {'NOT_SERVICE' | boolean}
+ */
+function pullServiceAccess ({ authentication: { item: user }, operation, fields, originalInput, listKey }) {
+    if (!user.permissions) return 'NOT_SERVICE'
+    if (!user.permissions.lists[listKey]) return false
+    /*
+        Example of listConfig:
+    lists: {
+        Organization: {
+            fields: {
+                __label__: 'ru',
+                country: 'ru',
+            }
+            access: 'ru',
+            where: {
+                id_in: ['uud1', 'uuid2']
+            }
+        }
+    }
+    custom: {
+        inviteNewOrganizationEmployee: true
+    }
+    */
+    const listPermissions = get(user, ['permissions', 'lists', listKey])
+    if (!listPermissions) return false
+    
+    const { access, where } = listPermissions
+    // If operation restricted on list level, we do not perform this request at all
+    if (!access[operation]) return false
+
+    return where || true    
```

Comment:
Why true by default?

Reply:
Because it will be used next in list accesses. 
Like 
```js
function canReadOrganizations(blahblak) {
    const serviceAccess = pullServiceAccess(...arguments)
    // buisnes logix next
}
```